src/sentry/api/serializers/models/organization_member.py

```diff
@@ -192,3 +193,25 @@ def serialize(
         d = super().serialize(obj, attrs, user)
         d["projects"] = attrs.get("projects", [])
         return d
+
+
+class OrganizationMemberSCIMSerializer(Serializer):  # type: ignore
+    def __init__(self, expand: Optional[Sequence[str]] = None) -> None:
+        self.expand = expand or []
+
+    def serialize(
+        self, obj: OrganizationMember, attrs: Mapping[str, Any], user: Any, **kwargs: Any
+    ) -> MutableMapping[str, JSONData]:
+
+        d = {
+            "schemas": [SCIM_SCHEMA_USER],
+            "id": obj.id,
+            "userName": obj.get_email(),  # TODO: does this get weird with secondary emails?
```

Comment:
> does this get weird with secondary emails?

My money would be on "yes". Furthermore, does this create edge cases for `OrganizationMember`s representing an unaccepted invite?

For reference:
```python
class OrganizationMember(Model):
    def get_email(self):
        if self.user_id and self.user.email:
            return self.user.email
        return self.email
```

Reply:
yeah, unfortunately there are a bunch of weird cases with secondary emails, although i think that is independent of scim. and I believe the answer is no on the second question -- or what type of edge case did you have in mind?

Reply:
Mainly the case where `self.user_id` is `None`, but I don't know offhand whether that's applicable.

Reply:
if the user_id is none, then it will default to the OrgMember's email which is the invite email -- think that is okay behavior.